Clarify the default logging category specifier (#30903)
dotnet/docs: docs/core/extensions/logging.md

In the preceding JSON:

- The `"Default"`, `"Microsoft"`, and `"Microsoft.Hosting.Lifetime"` log level categories are specified.

> - The `"Default"` value is applied to all categories that are not otherwise specified, effectively making all deafult value for all categories `"Information"`. This can be overridden by specifying a value for a category.

Reviewer: ```suggestion
- The `"Default"` value is applied to all categories that aren't otherwise specified, effectively making all default values for all categories `"Information"`. You can override this behavior by specifying a value for a category.
```
